jack-core/src/com/rapleaf/jack/queries/ModelQuery.java

```diff
@@ -1,10 +1,6 @@
 package com.rapleaf.jack.queries;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Nit-pick: I think our IntelliJ code formatting script would actually expand `*` to specific imports. Can you run that and keep the original import style?